testing/tests/DevExpress.exporter/exceljsParts/exceljs.tests.js

```diff
@@ -6477,6 +6478,72 @@ QUnit.module('_getFullOptions', moduleConfig, () => {
     });
 });
 
+QUnit.module('Deprecated warnings', moduleConfig, () => {
+    QUnit.test('CustomizeCell handler - warnings when \'cell\' field is not used', function(assert) {
+        assert.expect(3);
+        const stub = sinon.stub(errors, 'log', () => {
+            assert.strictEqual(true, false, 'error.log should not be called');
+        });
+
+        const done = assert.async();
+        const ds = [{ f1: 'f1_1' }];
+
+        const dataGrid = $('#dataGrid').dxDataGrid({
```

Comment:
testing/tests/DevExpress.exporter/exceljsParts/exceljs.tests.js - will we rename this file to something like '\dataGrid.exceljsParts\' or 'exceljs.datagrid.tests.js' ?

Reply:
Yes, we will rename this file in the future. We are going to create a separate commit for this purpose

Reply:
I renamed it